### PR TITLE
Add automated weekly runs of rivc64Linux

### PR DESCRIPTION
**Pull request by luhenry**

### CODE REVIEW

**Review by andrew-m-leonard — APPROVED** [review]
lgtm
Once merged it should trigger the job regeneration

File: pipelines/jobs/configurations/jdk20_pipeline_config.groovy [hunk 1]
```diff
@@ -164,7 +164,7 @@ class Config20 {
                         'temurin'   : '--create-jre-image --create-sbom'
                 ],
                 test                : [
-                        nightly: ['sanity.openjdk'],
+                        // nightly: ['sanity.openjdk'],
```
Comment (karianna) [inline pipelines/jobs/configurations/jdk20_pipeline_config.groovy:167]: prefer not to have commented out code...

**Review by karianna — CHANGES_REQUESTED** [review]

File: pipelines/jobs/configurations/jdk20_pipeline_config.groovy (hunk 1 above)
Comment (luhenry) [reply pipelines/jobs/configurations/jdk20_pipeline_config.groovy]: Fixed.

**Review by sxa — APPROVED** [review]
Very glad to have these back in the regular CI :-D

**Review by karianna — APPROVED** [review]
LGTM

**Pull request merged**